### ISSUE TITLE
how to increase intial amount

### ISSUE THREAD

**Issue by pamarthisai** [tudor199/Web3Arbitrage#13]
hi thanks for ur greater response i have one final doubt how to increase intial amount is thare any chance

**Comment by tudor199** [comment]
Hello!
Initial amount is not defined because you don`t need any money (beside for gas fees) to start the bot.
You can define the minimum profit for a specific token as in the readme.

**Comment by pamarthisai** [comment]
yes but bot taking very small amount of initial amount if we define bigger there is any chance because iam researching on flash loan bots from the past 3 years and ur bot no 1 in my research thats why iam trying to improve this bot from last three months pls help with possible suggition it is very usefull to my life i almost lost every thing in crypto. my only final hope is your bot only pls help me pls

**Comment by tudor199** [comment]
Bigger the amount, bigger the price impact is, the bots calculate the exact amount, in that way after a successful execution both pools would have the same exchange rate.
